Fix package imports and update MathJax version

Harriethw opened the pull request:
[Trello](https://trello.com/c/FnjQZn9Z/)

We want to display MathJax as SVG's in the mobile app. 

To do that, we need to use this external library - `futurelearn:rm-hw-mathjax-mobile` - with a slight modification to how it imports certain packages. We have therefore forked the repo and made the necessary changes. We decided forking the repo made more sense than vendoring it because we can easily point to this github repo in our package.json, it's easier to keep it in synch with the original repo, and we could potentially make a PR to the original repo with these fixes. 

### summary of changes

Previously the packages for extensions to Latex were not imported correctly, so we were not able to render [characters/commands](https://docs.mathjax.org/en/v2.7-latest/tex.html#supported-latex-commands) from the
different extension libraries to LaTex like [ams](http://docs.mathjax.org/en/latest/input/tex/extensions/index.html).

As we are making a change to the library anyway, we are taking the opportunity to update the version of MathJax that is vendored in the library.

sebjacobs commented:
🤔 It looks like MathJax 3.2.0 was released yesterday (https://github.com/mathjax/MathJax-src/releases/tag/3.2.0).

Harriethw committed:
Upgrade vendored Mathjax version to 3.2.0
Trello:https://trello.com/c/FnjQZn9Z/

We want to display Mathjax as SVG's in the mobile app. To do that, we
need this library with a slight modification to how it imports certain packages [1]. To that end we have forked the repo. As we are making a change to the library anyway, we are taking the opportunity to update the version of MathJax that is vendored.

In order to upgrade and re-vendor Mathjax-src, we followed Seb's previous work[2]

1. checked out the master branch from https://github.com/mathjaxMathJax-src/
2. npm install && npm run compile
3. cleaned the mathjax/es5/js directory (fromreact-native-mathjax-svg)
4. copied MathJax-src/js/ to react-native-mathjax-svg/mathjax/es5/js

[1] futurelearn/mobile@ea7418e
[2] futurelearn/mobile@d88dc37

Co-authored-by: roberta <roberta.mataityte@futurelearn.com>

Harriethw committed:
Fix import of MathJax packages

Trello:https://trello.com/c/FnjQZn9Z/

We want to display Mathjax as SVG's in the mobile app. To do that, we
need this library with a slight modification to how it imports certain packages.

Previously we were not able to render characters/commands[1] from the
different extension libraries to MathJax like `ams`[2] because they were not being imported correctly. This commit fixes that.

[1] https://docs.mathjax.org/en/v2.7-latest/tex.html#supported-latex-commands
[2] http://docs.mathjax.org/en/latest/input/tex/extensions/index.html

Co-authored-by: roberta <roberta.mataityte@futurelearn.com>

Harriethw commented:
Closing this as the remote repo has taken on board our changes, but with MathJax version 3.1.4. Keeping our fork open as we may experiment with MathJax 3.2!
